tests/10-migration-backend-tests/verification/verify_migrations_test.go

```diff
@@ -1210,7 +1217,7 @@ func Test_VerifyDuplicateMediaAndFile(t *testing.T) {
 
 		// (while we're ranging over the response data, resolve the file entities)
 		file := model.JsonApiFile{}
-		res.JsonApiData[i].JsonApiRelationships.File.Data.Resolve(t, &file)
+		res.JsonApiData[i].JsonApiRelationships.File.Data.ResolveWithBasicAuth(t, &file, drupalAdmin, drupalPass)
```

Comment:
These changes to use basic auth make sense based on the new behavior exhibited by Drupal with respect to the Admin user owning files.